src/services/contactService.ts

```diff
@@ -1,53 +1,153 @@
 import {RemoveContactArgs} from '@sphereon/ssi-sdk.contact-manager';
-import {Party, Identity} from '@sphereon/ssi-sdk.data-store';
-import Debug, {Debugger} from 'debug';
-
-import {APP_ID} from '../@config/constants';
-import {cmAddContact, cmAddIdentity, cmGetContacts, cmRemoveContact, cmUpdateContact} from '../agent';
+import {Identity, Party} from '@sphereon/ssi-sdk.data-store';
+import {ActionType, EventLogger, EventLoggerBuilder, InitiatorType, LoggingEventType, LogLevel, SubSystem, System} from '@sphereon/ssi-sdk.core';
+import {agentContext, cmAddContact, cmAddIdentity, cmGetContacts, cmRemoveContact, cmUpdateContact} from '../agent';
 import {IAddIdentityArgs, ICreateContactArgs, IGetContactsArgs, IUpdateContactArgs} from '../types';
 
-const debug: Debugger = Debug(`${APP_ID}:contactService`);
+const logger: EventLogger = new EventLoggerBuilder()
+  .withContext(agentContext)
+  .withLogLevel(LogLevel.INFO)
+  .withSystem(System.CONTACT)
+  .withSubSystem(SubSystem.CONTACT_MANAGER)
+  .withInitiatorType(InitiatorType.SYSTEM)
```

Comment:
You could change this to User, given most below methods are user initiated

Reply:
changed to USER